providers/fcontext.rb

```diff
@@ -38,7 +38,7 @@ def semanage_options(file_type)
 
 # Run restorecon to fix label
 action :relabel do
-  res = shell_out!('restorecon', '-irv', new_resource.file_spec)
+  res = shell_out!("find / -regextype posix-egrep -regex '#{new_resource.file_spec}' -execdir restorecon -iRv {} \\;")
```

Comment:
One last suggestion - to avoid any chance of commas in regex breaking this, convert to the comma way of shell_out:
`('find','/','-regextype','posix-egrep','-regex',new_resource.file_spec,'-execdir','restorecon','-iRv', '{}', ';')`
There might be a more elegant way to express that (e.g. `%w{} + [new_resource.file_spec] + %w{}`)
This is just to bullet-proof the interpolation here